charts/shoot-core/components/charts/kube-proxy/templates/kube-proxy-daemonset.yaml

```diff
@@ -45,6 +45,24 @@ spec:
           /sbin/sysctl -w net.ipv6.conf.all.disable_ipv6=1 || echo "IPv6 is already disabled. Doing nothing..."
         securityContext:
           privileged: true
+{{- end }}
+{{- if (semverCompare ">= 1.16" .Values.kubernetesVersion) }}
+      initContainers:
```

Comment:
`initContainers` key is already defined in [line 35](https://github.com/gardener/gardener/pull/2238/files#diff-0d19dce54b9a94959c40888d94978cb7R35).

Reply:
but only if this condition is true `{{- if and .Values.enableIPVS (semverCompare "< 1.14" .Values.kubernetesVersion) }}`